### DISCUSSION TITLE
Osu! Lazer HP drain for streams are too weak

### DISCUSSION THREAD

**Discussion by Tonylowkey** [ppy/osu#17912] [Ideas]
My osu has been down and for the first time i've been mainly using lazer where I've noticed a bad change for gameplay.

Note locking adjustment was a great idea but the hp drain for when I make a mistake or miss seems like im cheating with little to no effect on my health. This makes 7*-9* stream songs that I couldn't pass feel suprisingly easy to pass and not as satisfying to complete at all.

I know it's still in beta so this is just a suggestion but seeing some slightly stronger drain for stream mistakes will be much more understandable then a 10% difference in hp for hiting green and blue timing while missing almost half notes.

Here is evidence of how drain doesn't do much for mistakes. It feels more like if I miss on the slow parts I lose which i'm guessing is the time between notes balance made for notelocking that could possibly be adjusted.
video:
https://www.dropbox.com/s/u1o53q0oui74zu1/2022-04-21%2011-23-16.mkv?dl=0

**Comment by frenzibyte** [comment]
Maybe better to include as part of https://github.com/ppy/osu/issues/7975, since both of these focusing on the HP drain not matching stable? Although that thread mentions that osu!(lazer)'s HP drain implementation is rather harsher than softer.

Or well, since this focuses on streams, probably better leaving it as-is here.

**Reply by Tonylowkey** [reply]
It actually could be just on opposite ends of the same issue so adjusting this could affect that and what not so there is a strange dilemma there.

**Comment by jedel1043** [comment]
I have been playing on lazer for a couple of months, and I've just come to realize that the harshness of the HP drain depends on the number of total hit objects.

For example, https://osu.ppy.sh/beatmapsets/859515#osu/2105741, A stream map with HP 6, is easily passable by spamming, but if you make a "training" difficulty with only the last 30 seconds of the song or so, the map becomes pretty much only passable by FCing.

Should I open an issue for this?

**Reply by mk56-spn** [reply]
hp is an ongoing issue that has been discussed in many discussion, i dont think the need for an issue has arisen yet, since it tends to be done ( if at all, many things go from discussion to pr directly) after how its going to be adressed is clearer, and even if it will be
